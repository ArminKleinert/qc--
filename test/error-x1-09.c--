
register bits8 c;

section "data" 
{
    x:
    bits16;

}

foo (bits32 y) {

    y =  c + bits32[x];
}




