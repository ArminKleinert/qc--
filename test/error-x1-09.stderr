Character 88-101 Error: type error in application
